### PR TITLE
Output will not show parameters with null values.

### PR DESCRIPTION
**Pull request by chilipadiboy**
#40 

Todo:
fix tests so that remark is not expected to show up

### CODE REVIEW

File: src/main/java/seedu/address/model/person/ReadOnlyPerson.java [hunk 1]
```diff
@@ -64,21 +64,35 @@ default boolean isSameStateAs(ReadOnlyPerson other) {
...
-                .append(" Birthday: ")
-                .append(getBirthday())
-                .append(" Remarks: ")
-                .append(getRemark())
-                .append(" Website: ")
-                .append(getWebsite())
-                .append(" Tags: ");
-        getTags().forEach(builder::append);
+        builder.append(getName());
+        if (!(getPhone().toString() == null)) {
```
Comment (Jemereny) [inline src/main/java/seedu/address/model/person/ReadOnlyPerson.java]: use != null. Same for the ones below

**Review by Jemereny — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/person/ReadOnlyPerson.java (hunk 1 above)
Comment (chilipadiboy) [reply src/main/java/seedu/address/model/person/ReadOnlyPerson.java]: aight

**Review by Jemereny — APPROVED** [review]

**Comment by Jemereny** [comment]
LGTM

**Comment by chilipadiboy** [comment]
Fixes #40

**Pull request merged**